analyzers/tests/SonarAnalyzer.UnitTest/Rules/CollectionEmptinessCheckingTest.cs

```diff
@@ -30,10 +30,25 @@ public class CollectionEmptinessCheckingTest
     {
         [TestMethod]
         public void CollectionEmptinessChecking_CS() =>
-            OldVerifier.VerifyAnalyzer(@"TestCases\CollectionEmptinessChecking.cs", new CS.CollectionEmptinessChecking());
+            new VerifierBuilder()
+                .AddAnalyzer(() => new CS.CollectionEmptinessChecking())
+                .AddPaths("CollectionEmptinessChecking.cs")
+                .Verify();
 
         [TestMethod]
         public void CollectionEmptinessChecking_VB() =>
-            OldVerifier.VerifyAnalyzer(@"TestCases\CollectionEmptinessChecking.vb", new VB.CollectionEmptinessChecking());
+            new VerifierBuilder()
+                .AddAnalyzer(() => new VB.CollectionEmptinessChecking())
+                .AddPaths("CollectionEmptinessChecking.vb")
+                .Verify();
+
+        [TestMethod]
+        public void CollectionEmptinessChecking_CodeFix_CS() =>
+            new VerifierBuilder()
+                .AddAnalyzer(() => new CS.CollectionEmptinessChecking())
+                .AddPaths("CollectionEmptinessChecking.cs")
+                .WithCodeFix<CS.CollectionEmptinessCheckingCodeFix>()
+                .WithCodeFixedPath(@"TestCases\CollectionEmptinessChecking.Fixed.cs")
```

Comment:
This problem was already fixed on our master
```suggestion
                .WithCodeFixedPath(@"CollectionEmptinessChecking.Fixed.cs")
```